build.gradle.kts

```diff
@@ -580,19 +590,6 @@ tasks {
 
   test { dependsOn(project.tasks.getByPath("buildCody")) }
 
-  configurations {
```

Comment:
Was that not needed?

Reply:
It looks so. the test are passing without it. Note that configuration creation is redundant. SourceSets persist.